src/openms_gui/source/VISUAL/DataSelectionTabs.cpp

```diff
@@ -64,6 +64,7 @@ namespace OpenMS
     spectra_view_widget_(new SpectraTreeTab(this)),
     id_view_widget_(new SpectraIDViewTab(Param(), this)),
     dia_widget_(new DIATreeTab(this)),
+    tab_ptrs_{ spectra_view_widget_, id_view_widget_, dia_widget_ },   // make sure to add new tabs here!
```

Comment:
Might also be better to have this generated based on the data stored in the workspace / the open window but well this would be quite some effort

Reply:
you mean across all layers?
This would break a central concept, but maybe its worth thinking about where this would lead.